test/support/i18n.js

```diff
@@ -0,0 +1,17 @@
+import IntlMessageFormat from 'intl-messageformat';
+import memoize from 'lodash/memoize';
+import messages from '../../i18n/en-US';
+
+const getFormatter = memoize(messageId => new IntlMessageFormat(messages[messageId], 'en-US'));
+
+/**
+ *
+ * @param {string} messageId id of message in i18n file
+ * @param {Object} variables substitutions for translation, e.g. counts for plurals
+ * @example cy.getByText(localize('be.greeting', {name: 'Alex'})).click()
+ */
+const localize = (messageId, variables = {}) => {
+    return getFormatter(messageId).format(variables);
+};
+
+export { localize, localize as l };
```

Comment:
I'd like to avoid importing this utility in every test. Can you move it to a global method or add a custom Cypress command, such as `Cypress.Commands.add('l', localize);`? Let's also standardize on using the `l` alias everywhere and not expose `localize`.

Reply:
Tried to add it as a command but it turns out cy commands aren't functions you can just use inline, they need to be chainable

Reply:
Also there doesn't appear to be a way to set a global other than setting a value on window